public/css/restaurants.css

```diff
@@ -0,0 +1,184 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#F2F2F2;
+    --food-slider-bg:#171A29;
+    --food-top-brands:#F8F8F8;
+    --bg-white:#F5F5F5;
+    --border-black-op: rgba(0,0,0,0.1);
+    --tab-border-bottom:#6200EE;
+    --tab-ripples:#E9DDF9;
+     --tab-font:#6200EE;
+     --radio-color:#4f46e5;
+     --border:rgb(0,0,0,0.2);
+
+}
+.nav-zindex{
+    z-index: 999;
+}
+
+body{
+    background: var(--bg-white);
+
+}
+
+.navbar-bg{
+    padding: 1rem 1rem 0;
+    border-bottom: 1px solid var(--border-black-op);
+
+  }
+
+.food-search-container{
+    height: 50vh;
+    background-image: url('../img/background/food-banner.jpg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding-top: 3rem;
+}
+
+.banner-head-txt{
+    font-size: 4.2rem;
+}
+.food-banner-text{
+    font-size: 2rem;
+    font-weight: 500;
+}
+.input-w{
+    width: 36%;
+}
+
+.input{
+    outline: none;
+    border: 0px;
+}
+
+.restro-lists-container{
+    width: 95%;
+    margin: auto;
+}
+
+.card-border{
+    border: 1px solid rgb(0,0,0,0.0);
```

Comment:
if color is not needed you can remove rgb or just give transparent

Reply:
done